src/app/public/modules/inline-form/inline-form.component.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../styles/mixins";
+
+.sky-inline-form {
+  background: $sky-background-color-neutral-light;
+  border: 1px solid $sky-border-color-neutral-medium;
+  padding: $sky-padding-even-default;
+  width: 100%;
+
+  .sky-inline-form-footer {
+    margin-top: $sky-margin-stacked-default;
```

Comment:
looking at this, i think we need to use `$sky-margin-stacked-separate` instead

Reply:
Done.